schemas/iso19139/src/main/plugin/iso19139/formatter/jsonld/iso19139-to-jsonld.xsl

```diff
@@ -202,7 +202,7 @@
     <xsl:if test="position() != last()">,</xsl:if>
     </xsl:for-each>],
     "provider" : [<xsl:for-each select="gmd:identificationInfo/*/gmd:pointOfContact/*[gmd:role/gmd:CI_RoleCode/@codeListValue='resourceProvider' or gmd:role/gmd:CI_RoleCode/@codeListValue='custodian']">
-    <xsl:apply-templates mode="doContact" select="."/>
+    <xsl:apply-templates mode="doContact" select="."/>,
```

Comment:
The code should be checking if there are `gmd:contact` elements or not to add the `,`. To test:

```
<xsl:if test="count(//gmd:contact) > 0">,</xsl:if>
```

Reply:
Reverted the change, apologies for the confusion as `gmd:contact` is mandatory, so the change proposed was not required.